apps/meteor/app/api/server/v1/chat.ts

```diff
@@ -100,17 +101,20 @@ API.v1.addRoute(
 				...(type && { type }),
 			};
 
-			const result = await Meteor.callAsync('messages/get', roomId, getMessagesQuery);
+			const result = await getMessageHistory(roomId, this.userId, getMessagesQuery);
 
 			if (!result) {
 				return API.v1.failure();
 			}
+			if (typeof result === 'boolean') {
+				return API.v1.success({ result: {} });
+			}
```

Comment:
maybe instead of `boolean`, this one could return `false` so the check above can cover it and return failure instead? would that be a breaking change? would this one be a breaking change? 👀